rcv/rcv_utils.py

```diff
@@ -0,0 +1,17 @@
+import configparser
+import cv2
+import logging
+import logging.config
+import time
+
```

Comment:
pretty sure it doesn't need to import cv2 and time

Reply:
O.K.